### ISSUE TITLE
How to send a command back into talon from a variable?

### ISSUE THREAD

**Issue by Madd0g** [talonvoice/talon#107]
I'm playing with `phrase_history.py`, I'm wondering how from knowing a phrase as a string, I can send it back into talon as a command?  

For example, suppose I want to say "history 1" and I get "made crunch drum sit red" as a string, is there a way to feed the phrase back into talon as a command?

Thanks

**Comment by landonwilkins** [comment]
Possibly related Q&A:

> Q: How to test a voice command in the repl (`~/.talon/bin/repl`)?

> A: `engine.mimic("phrase here".split())`

**Comment by Madd0g** [comment]
I tried using `engine.mimic` before (with the built-in engine and not from the REPL) and it often printed the phrase (or part of it) into the app that was focused. 

I'm not sure if it always happened or only when I was holding keys while running `engine.mimic`, but I was always wary of using it. I just tried it right now from the REPL and it works perfectly. I'll try using it from python code again. Thanks

EDIT: works perfectly
